src/accordion/accordion.ts

```diff
@@ -199,6 +207,14 @@ export class NgbAccordion implements AfterContentChecked {
    */
   isOpen(panelId: string): boolean { return this._states.get(panelId); }
 
+  /**
+   * @internal
```

Comment:
Are you sure about `@internal`? I believe AOT has problems with `private` methods/fields used in the template. Note then that `isOpen` also has `@internal` comment.

About inline the logic I completely agree but there's currenctly a strange behaviour where `null` is converted into `false` and the attribute is the added `hidden="false"`, which is selected by Boostrap CSS selector `[hidden] {...}`.
I found a similar [issue](https://github.com/angular/angular/issues/8960).